src/worker/tasks/sendWebhookWorker.ts

```diff
@@ -52,7 +70,7 @@ const handler: Processor<any, void, string> = async (job: Job<string>) => {
 let _worker: Worker | null = null;
 if (redis) {
   _worker = new Worker(SEND_WEBHOOK_QUEUE_NAME, handler, {
-    concurrency: 1,
+    concurrency: 10,
```

Comment:
Increase concurrency now that there's more webhooks to send. Webhook calls are independent.